yarn.lock

```diff
@@ -17024,7 +17024,7 @@ drange@^1.0.2:
   resolved "https://registry.yarnpkg.com/drange/-/drange-1.1.1.tgz#b2aecec2aab82fcef11dbbd7b9e32b83f8f6c0b8"
   integrity sha512-pYxfDYpued//QpnLIm4Avk7rsNtAtQkUES2cwAYSvD/wd2pKD71gN2Ebj3e7klzXwjocvE8c5vx/1fxwpqmSxA==
 
-dset@^3.1.2, dset@^3.1.4:
+dset@^3.1.2:
```

Comment:
it's not really "removed" if it's still in the yarn lock. If you remove it and run `yarn install` does it stay gone?

Reply:
I did run `yarn install` after removing it from resolution but it looks like that other version `dset@^3.1.2` is required by `@graphql-tools/utils`